[DCK] Increase worker memory by default

yajo opened the pull request:
HACK https://github.com/odoo/odoo/issues/30350

Odoo's ORM just takes forever in some queries if your worker has not enough memory.

To avoid hitting that problem, let's configure a big memory default for Doodba users.

pedrobaeza commented:
Can you overwrite that value by config of pgtune? Or the only way to tune PG is through this?

yajo commented:
> config of pgtune?

What do you mean by this?

pedrobaeza commented:
pgtune is the tool for telling an optimized configuration for Postgres according specifications (well, I think you already know this). The question is: how do you put this config right now for an scaffolding?

yajo commented:
Oh I see.

Yes, this is the place to configure postgres, and you can also put there the optimizations made by pgtune.

In any case, for this specific scenario, pgtune values usually will not work too (I tested that already), because pgtune "tunes" your postgres for _good_ usage, but the main point is that Odoo ORM is using _bad_ queries, so no tool autooptimizes for that... :confused:

pedrobaeza commented:
OK, I'm not so sure that query is so bad. We have been with it since a lot, and systems with more records work properly. Any way, it's out of scope of this. Let's move on here when CI goes well.

yajo committed:
[DCK] Increase worker memory by default

HACK https://github.com/odoo/odoo/issues/30350

Odoo's ORM just takes forever in some queries if your worker has not enough memory.

To avoid hitting that problem, let's configure a big memory default for Doodba users.
